### ISSUE TITLE
Bugs in the ldm training and visualization

### ISSUE THREAD

**Issue by XuanchaoPENG** [jojo23333/LatetHOI#9]
Thanks for your work.
When I train the ldm model, the loss and val_loss seem to be correct. However, when I run the visualization script, I got the following result:

<img width="527" height="462" alt="Image" src="https://github.com/user-attachments/assets/822c08c4-0043-4d15-8e0d-cc05c185940c" />

The object doesn't move and hands are shaking. The root of hands are static. Any possible solutions?

**Comment by lingyin2424** [comment]
hello，i also run vis script, but only got grey mesh. I want to know how to apply texture？

**Comment by Yeelj** [comment]
@XuanchaoPENG I also encountered this problem, how did you solve it?
